processor/resourcedetectionprocessor/internal/env/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+type: env
```

Comment:
I don't think the `env` detector should have a metadata.yaml file

Reply:
Ok, that makes sense. I brought back the `env` detector to its initial state